panel/1.0/webserver_configuration.md

```diff
@@ -85,3 +85,5 @@ systemctl restart apache2
 
 :::
 ::::
+
+#### Next Step: [Wings Installation](../../wings/./installing.md)
```

Comment:
why the random dot?

Reply:
Looks like my vscode auto-suggest filled that in, but it works even with the dot as I tested prior to the PR

Removed the random dot